src/app/ApplicationTemplate.Shared.Views/Content/Settings/SettingsPage.xaml

```diff
@@ -14,7 +17,8 @@
 		<CommandBar Content="Profile"
 					x:Uid="Settings_CommandBar" />
 
-		<ScrollViewer Grid.Row="1">
+		<ScrollViewer Grid.Row="1"
```

Comment:
for some unknown reason to me at the moment I'm writing this, it's not working in this page (at least android)

Reply:
Any update on this? Did you try to update Uno?

Reply:
No updates really. At the time, I wanted to bring the whole logic with did on projects, but it was causing me too much problem so I've put it on the side. And I don't think I'll have time for this for a while...

Reply:
At this point, I could probably abandon this PR and probably find a better approach for the menu in the future

Reply:
I've closed it and deleted the branch